Presentation.Web/Controllers/API/ItSystemUsageDataSensitivityLevelController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Core.ApplicationServices.SystemUsage;
+using Core.DomainModel.ItSystemUsage.GDPR;
+using Core.DomainModel.Result;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.ItSystemUsage;
+
+namespace Presentation.Web.Controllers.API
+{
+    [InternalApi]
+    [RoutePrefix("api/v1/itsystemusage")]
+    public class ItSystemUsageDataSensitivityLevelController : BaseApiController
+    {
+
+        private readonly IItSystemUsageService _usageService;
+
+        public ItSystemUsageDataSensitivityLevelController(IItSystemUsageService usageService)
+        {
+            _usageService = usageService;
+        }
+
+        /// <summary>
+        /// Opretter en ny systemrelation
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="dataSensitivityLevel"></param>
+        /// <returns></returns>
+        [HttpPatch]
+        [Route("{id}/sensitivityLevel/add")]
+        public HttpResponseMessage AddSensitivityLevel(int id, [FromBody]SensitiveDataLevel dataSensitivityLevel)
+        {
+            var result = _usageService.AddSensitiveDataLevel(id, dataSensitivityLevel);
+
+            return result.Match
+            (
+                onSuccess: itSystemUsageDataLevel =>  Ok(MapSensitiveDataLevelDTO(itSystemUsageDataLevel)),
+                onFailure: operationError =>
+                {
+                    switch (operationError.FailureType)
+                    {
+                        case OperationFailure.NotFound:
```

Comment:
@JacobMalling  nej det var ikke det her vi talte om. Her ved du ikke hvor den kommer fra. Hvis man får notfound her er det jo 404 (it system anvendelse findes ikke).. hvis du står i servicen og får notfound tilbage fra den kommando på den fundne anvendelse så er det en "badInput" og det tolkes til badrequest heroppe.. så.. forfra :-)